app/scripts/modules/core/delivery/service/executions.transformer.service.js

```diff
@@ -299,7 +299,7 @@ module.exports = angular.module('spinnaker.core.delivery.executionTransformer.se
         let jenkins = deploymentDetails[0].jenkins;
         execution.buildInfo = {
           number: jenkins.number,
-          url: `${jenkins.host}job/${jenkins.name}/${jenkins.number}`
+          url: deploymentDetails[0].buildInfoUrl || `${jenkins.host}job/${jenkins.name}/${jenkins.number}`
```

Comment:
This makes sense to me. What about the build number itself in the text of the link (the line above)? Does that need to be defaulted to something else?